src/content/docs/infrastructure/host-integrations/host-integrations-list/temporal-monitoring-integration.mdx

```diff
@@ -48,6 +49,9 @@ you need to add the prometheus endpoint and port in the `docker-compose.yml` fil
 ```yml
 sudo nano docker-compose/docker-compose.yml
 ```
+- Below the **container_name**: **temporal** in the Environment section, include the Prometheus endpoint as follows: `- PROMETHEUS_ENDPOINT=0.0.0.0:8000`.
```

Comment:
@pkudikyala if `container_name: temporal` is a line in the file, this should be inside backticks instead of bold.

Reply:
Hi, @rhetoric101 , 
Thank you for your feedback. I've addressed your comments. Kindly review it.